config/hubs/2i2c.cluster.yaml

```diff
@@ -47,12 +47,11 @@ hubs:
         hub:
           config:
             Authenticator:
-              allowed_users: &staging_users
-                # this will be replaced with the actual list of
-                # staff google ids by the deployer script
+              # this will be replaced with the actual list of
+              # staff google ids by the deployer script
+              admin_users: &staging_admins
                 - <staff_google_ids>
-                - colliand@gmail.com
-              admin_users: *staging_users
+              username_pattern: '^(.+@2i2c\.org|.+@berkeley\.edu|deployment-service-check)$'
```

Comment:
Do you think we should test/show off the CILogon feature on the staging hub or it's best if I deploy a new hub?

Reply:
I would be OK to have it on staging, @2i2c-org/tech-team WDYT?

Reply:
I actually feel like we should have a dedicated hub to show off these kinds of things - the staging hub is meant for staging and trying out changes as part of our deployment pipeline, so I feel like we should reserve it for those purposes rather than showing off new features and such. What if we deployed a hub like `demo.2i2c.cloud` that was for demonstration purposes, and where we could change the config etc without worrying that it'll affect our other deployments?

Reply:
What if we used the cloudbank demo hub? I don't think that's used right now
https://github.com/2i2c-org/infrastructure/blob/e8e80aeb4b9955f86bf9ca4d0a01d4ff1ff1ff58/config/hubs/cloudbank.cluster.yaml#L189